pkg/edition/java/lite/forward.go

```diff
@@ -62,6 +93,19 @@ func Forward(
 	pipe(log, src, dst)
 }
 
+// reference to the current lite mode config
+var currentConfig *config.Config
+
+// SetConfig sets the current lite mode config
+func SetConfig(cfg *config.Config) {
+	currentConfig = cfg
+}
+
+// getConfig returns the current lite mode config
+func getConfig() *config.Config {
+	return currentConfig
+}
+
```

Comment:
This is a cheeky hack to pass state. It should be handled differently.
I don't want this :)